### PR TITLE
Update Purchaser/Customer related naming -> Rename of old API before adding in new API.

### PR DESCRIPTION
**Pull request by taquitos**
`invalidatePurchaserInfoCache -> invalidateCustomerInfoCache`
`getPurchaserInfo -> getCustomerInfo`
Also update comments.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ yarn.lock
 examples/cordova-sample/MyApp/node_modules
 examples/cordova-sample/MyApp/platforms
 examples/cordova-sample/MyApp/plugins
+package-lock.json
```
Comment (taquitos) [inline .gitignore:11]: I keep having to ignore this, it's generated each time, so let's ignore it officially.

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,5 @@ yarn.lock
...
+*.bck
```
Comment (taquitos) [inline .gitignore:12]: Same here, super annoying to have to ignore it.

File: examples/cordova-sample/MyApp/www/js/index.js [hunk 3]
```diff
@@ -82,6 +82,8 @@ const app = {
         }
     );
 
+    Purchases.invalidateCustomerInfoCache();
```
Comment (taquitos) [inline examples/cordova-sample/MyApp/www/js/index.js:85]: Added as a way to keep api testing while coding this.

Comment (aboedo) [reply examples/cordova-sample/MyApp/www/js/index.js:85]: maybe add that in as a comment?

**Review by aboedo — APPROVED** [review]

File: examples/cordova-sample/MyApp/www/js/index.js (hunk 3 above)
Comment (taquitos) [reply examples/cordova-sample/MyApp/www/js/index.js:86]: ✅

**Pull request merged**